build.rs

```diff
@@ -0,0 +1,108 @@
+extern crate regex;
```

Comment:
It'd probably be a good idea to run this once and check the generated code in. There's no real reason to go through the effort of doing this every time the crate's compiled.